### PR TITLE
Updating copy for the phase2 about page

### PR DESCRIPTION
**Pull request by leanthebean**
## Summary
Updating copy for the phase2 about page

## Testing Plan
Local

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[X] No
[ ] Yes
```

### CODE REVIEW

**Review by brekk — APPROVED** [review]

File: components/About/data.tsx [hunk 1]
```diff
@@ -101,48 +101,43 @@ export const callsToAction = {
...
       content:
-        'Hosting a node using your graffiti automatically earns you points. You will only be rewarded points for hosting one node.',
-      points: ['12 hours = 100 points'],
+        'You can earn points by hosting an Iron Fish full node. To do so, make sure your telemetry is turned on, and your graffiti is set. You will only be rewarded points for hosting one node per graffiti.',
+      points: ['12 hours of continuous uptime = 100 points'],
       status: Status.New,
-      earn: 0,
-      disabled: true,
+      ctaText: 'Set up instructions',
+      href: 'https://coda.io/d/_dngSKECR9gO/Phase-2-FAQ_suBOW',
```
Comment (dguenther) [inline components/About/data.tsx:108]: This doc doesn't appear to be public yet -- is it ready to open up?

File: components/About/data.tsx [hunk 2]
```diff
@@ -101,48 +101,43 @@ export const callsToAction = {
...
-      earn: 0,
       points: [
         'Small = 250 points',
         'Medium = 500 points',
         'Large = 1000 points',
       ],
       ctaText: 'Submit a PR',
-      // href: 'https://github.com/iron-fish/ironfish/pulls',
-      submissionForm: false,
+      href: 'https://github.com/iron-fish/ironfish/pulls',
```
Comment (dguenther) [inline components/About/data.tsx:120]: The text in this section mentions a form -- are we going to use the same form as last time? If so, we should switch this button to go to the form.

File: components/About/data.tsx [hunk 3]
```diff
@@ -101,48 +101,43 @@ export const callsToAction = {
...
       title: `Sending a transaction`,
       content: (
         <>
-          Sending your mined $IRON earns you points. Use command{' '}
+          For every transaction you send to the Iron Bank of over 0.1 coins, you
+          get 10 points. To make this easier, we made a command that
+          automatically sends this transaction for you:
           <pre className="bg-iflightgray inline p-1 font-favorit">
-            send_transaction
+            ironfish deposit
```
Comment (dguenther) [inline components/About/data.tsx:133]: I believe it's going to be `testnet:deposit`, but we can readjust the docs when that work is ready.

File: components/About/data.tsx [hunk 4]
```diff
@@ -101,48 +101,43 @@ export const callsToAction = {
...
-          . You get 1 point for each iron you send, as long as you send at least
-          1 iron.
         </>
       ),
-      points: ['1 $IRON sent = 1 point (minimum 1 $IRON)'],
-      ctaText: 'Get started with mining',
-      // href: 'https://ironfish.network/docs/onboarding/miner-iron-fish',
+      points: ['10 points for a transaction over 0.1 coins'],
+      ctaText: 'Learn more',
+      href: 'https://ironfish.network/docs/onboarding/miner-iron-fish',
```
Comment (dguenther) [inline components/About/data.tsx:139]: Button links to a doc page about mining, but the above text is about sending transactions using the `deposit` command -- maybe hiding the "Learn More" would be fine until we have something to put there?

**Review by dguenther — COMMENTED** [review]
Commented on some things I noticed, I don't think any of these have to be blocking

File: components/About/data.tsx (hunk 1 above)
Comment (leanthebean) [reply components/About/data.tsx:108]: yeah I'll change that

File: components/About/data.tsx (hunk 3 above)
Comment (leanthebean) [reply components/About/data.tsx:133]: double checked with Mat that is indeed `ironfish deposit`

File: components/About/data.tsx (hunk 2 above)
Comment (leanthebean) [reply components/About/data.tsx:120]: Updated

**Pull request merged**